### ISSUE TITLE
Crash when uploading photo

### ISSUE THREAD

**Issue by 3ananda** [FlutterFlow/flutterflow-issues#264]
## Expected behavior (required)

Upload the photo

## Current behavior (required)

When a photo from the gallery is selected the application crashes. This does not happen when selecting a video or taking a photo, only when selecting from the gallery.

## To Reproduce (required)

1. Create new page
2. Add upload media action.
3. Select media in IOS.

It happens in IOS

**Comment by agreaves** [comment]
Hi @3ananda, thanks for the report. I have not been able to reproduce this issue with these steps. I'm guessing it may have to do with the specific image being uploaded.

Can you please include a screen recording of the issue and the Bug Report Code for the project?

**Comment by 3ananda** [comment]
![Crash video](https://user-images.githubusercontent.com/109097781/209319349-3ea47490-192d-496e-8c9d-8dc690daebe9.gif)

**Comment by bulgariamitko** [comment]
In order to keep our issue tracker useful and maintainable for all users, please follow the bug report template while reporting bugs. We will be happy to re-assess when the necessary information is provided.
Thank you!

**Comment by 3ananda** [comment]
I think it's pretty obvious what's going on. It's very simple, no further explanations are needed: When you select a photo in the gallery the application crashes. It's simple. I don't know what else I can add.

**Comment by agreaves** [comment]
Hi @3ananda, the problem is that we have not been able to reproduce the issue with the steps you provided. Gallery image upload on iOS seems to continue to work. Which means it likely has to do with either your specific project, the specific image being uploaded, or the device you are using. Without any additional information about this there isn't much we can do on our end.

If you could provide a Bug Report Code for your project that would allow us to investigate whether there is anything going on in your specific project. If you are experiencing this even in a simple test project by following the steps you provided, a Bug Report Code for that would be even more useful.

**Comment by morad** [comment]
I have the problem that I've just reported on the Flutterflow.

When I download the App via testFlight, my app crashes when I try to upload a file.

I used to have the same problem on simulator as well, when I deploy it via Android Studio. But after I've updated Firebase SDK with "pod update" for iOS, the problem no longer exists on my local environment. 

I believe whatever Firebase version for iOS that flutterflow is using, it will need to be updated to the latest.
